driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkConsumer.java

```diff
@@ -47,11 +45,11 @@
     private final ExecutorService executor;
     private final Future<?> consumerTask;
     private volatile boolean closing = false;
-    private boolean autoCommit;
-    public KafkaBenchmarkConsumer(KafkaConsumer<String, byte[]> consumer, Properties consumerConfig, ConsumerCallback callback) {
+
+    public KafkaBenchmarkConsumer(KafkaConsumer<String, byte[]> consumer, ConsumerCallback callback) {
```

Comment:
This change is reverting #148. Is there any reason to do so?

Reply:
I'll need to discuss with @eolivelli

Reply:
I don't see any reason to revert this.

My understanding is that simply @dave2wave in our fork we did not have #148 
I would simply revert this change here in this PR

Reply:
Done